packages/app/screens/landing/SignUpScreen.js

```diff
@@ -0,0 +1,332 @@
+import React, { useCallback, useRef } from "react";
+import axios from "axios";
+import {
+  StyleSheet,
+  View,
+  Text,
+  KeyboardAvoidingView,
+  TouchableWithoutFeedback,
+  Keyboard,
+  Alert,
+  TouchableOpacity,
+  Platform,
+} from "react-native";
+
+import * as WebBrowser from "expo-web-browser";
+
+import Button from "@app/components/Button";
+import Screen from "@app/components/Screen";
+import TextInput from "@app/components/TextInput";
+import GoogleIcon from "@app/assets/google-icon";
+
+import { useUserContext } from "@app/utils/UserContext";
+import { ENDPOINT } from "@app/utils/constants";
+import { getUserInfo } from "@app/utils/datalayer";
+
+import EyeOff from "@app/assets/eye-line-off";
+import EyeOn from "@app/assets/eye-line-on";
+import * as SecureStore from "@app/utils/SecureStore";
+
+import Constants from "expo-constants";
+import DividerWithText from "@app/components/DividerWithText";
+
+WebBrowser.maybeCompleteAuthSession();
+
+function SignUpScreen({ navigation, route }) {
+  const { user, setUser } = useUserContext();
```

Comment:
Why do you get the user context here for the signup screen? It should be null at this point since the user is unauthenticated and registering for our app.